cookbook/validation/custom_constraint.rst

```diff
@@ -158,7 +158,7 @@ Constraint Validators with Dependencies
 If your constraint validator has dependencies, such as a database connection,
 it will need to be configured as a service in the Dependency Injection
 Container. This service must include the ``validator.constraint_validator``
-tag and may include an ``alias`` attribute:
+tag and *MUST* include an ``alias`` attribute:
 
```

Comment:
I'd add that `alias` must match what is returned by the `validateBy` method and that it must not necessarily match the service id